### PR TITLE
RD-6232 In  the detaills of the root node is not shown

### PR DESCRIPTION
**Pull request by leon-good-life**
## Description
Jira ticket: RD-6232

## Screenshots / Videos
Before:
Screenshot 2022-12-18 at 12 16 05

After:
Screenshot 2022-12-18 at 12 12 54

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [ ] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
`blueprint_sources_spec` after test update:
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/4152/pipeline

`blueprint_sources_spec`:
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/4151/pipeline

 

## Documentation
Nothing to add:
https://docs.cloudify.co/latest/working_with/console/widgets/blueprintsources/

### CODE REVIEW

File: widgets/blueprintSources/src/BlueprintSources.tsx [hunk 1]
```diff
@@ -95,7 +100,8 @@ const RightPane = ({
 
     return (
         <Center>
-            <Icon name="file outline" size="big" color="grey" />
+            <Bold>This is a folder</Bold>
+            Nothing to show.
```
Comment (kubama) [inline widgets/blueprintSources/src/BlueprintSources.tsx:104]: i18n should be used

Comment (leon-good-life) [reply widgets/blueprintSources/src/BlueprintSources.tsx]: I added translations: [73cc7bf](https://github.com/cloudify-cosmo/cloudify-stage/pull/2346/commits/73cc7bf90d8d5effc8e41611da0097802649c134)

**Comment by leon-good-life** [comment]
I re-run `blueprint_sources_spec `:
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/4185/pipeline

It is failing on Jenkins but passes locally

**Comment by leon-good-life** [comment]
Error of `blueprint_sources_spec` (only in Jenkins, passes locally):
Blueprint%20Sources%20widget%20--%20blueprint%20is%20not%20selected%20--%20before%20all%20hook%20(failed)

https://user-images.githubusercontent.com/17104212/208913921-31e6e18d-398d-4043-a8cb-8d6ba5132edf.mp4

**Review by kubama — APPROVED** [review]

**Comment by leon-good-life** [comment]
The test failed on master branch too and there is an open ticket for it: RD-6642
So I will merge this PR.

**Pull request merged**